Mon Nov 03 14:01:46 CET 2014 : Apache Derby Network Server - 10.10.1.3 - (1557168) démarré et prêt à accepter les connexions sur le port 1527
----------------------------------------------------------------
Mon Nov 03 14:01:47 CET 2014:
Amorçage de Derby version The Apache Software Foundation - Apache Derby - 10.10.1.3 - (1557168) : instance a816c00e-0149-75bf-37bf-fffff4e597bf 
dans le répertoire de base de données C:\Users\Thomas\Desktop\java_tp\GSBJava\nbproject\GSB avec le chargeur de classe sun.misc.Launcher$AppClassLoader@1d44bcfa
Chargé de file:/C:/Program%20Files/Java/jdk1.8.0_20/db/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_20-b26
user.dir=C:\Program Files\Java\jdk1.8.0_20\db
os.name=Windows 8.1
os.arch=amd64
os.version=6.3
derby.system.home=C:\Users\Thomas\Desktop\java_tp\GSBJava\nbproject
Chargeur de classe de la base de données démarré - derby.database.classpath = ''
----------------------------------------------------------------
Mon Nov 03 14:01:55 CET 2014:
Fermeture de l'instance a816c00e-0149-75bf-37bf-fffff4e597bf sur le répertoire de base de données C:\Users\Thomas\Desktop\java_tp\GSBJava\nbproject\GSB avec le chargeur de classe sun.misc.Launcher$AppClassLoader@1d44bcfa 
Mon Nov 03 14:01:55 CET 2014 Thread[DRDAConnThread_3,5,main] (DATABASE = GSB), (DRDAID = {2}), Arrêt de la base de données 'GSB'.
Mon Nov 03 15:32:39 CET 2014 Thread[DRDAConnThread_3,5,main] Cleanup action starting
java.sql.SQLException: Base de données 'GSB' introuvable.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
Caused by: java.sql.SQLException: Base de données 'GSB' introuvable.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 19 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Base de données 'GSB' introuvable.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
============= end nested exception, level (1) ===========
Cleanup action completed
Mon Nov 03 15:32:39 CET 2014 Thread[DRDAConnThread_3,5,main] (DATABASE = GSB), (DRDAID = {4}), Base de données 'GSB' introuvable.
----------------------------------------------------------------
Mon Nov 03 15:36:19 CET 2014 : fermeture du moteur Derby
----------------------------------------------------------------
